### DISCUSSION TITLE
Can we set specific cameras to on-demand only?

### DISCUSSION THREAD

**Discussion by thatdaveguy1** [mrlt8/docker-wyze-bridge#603] [Q&A]
I know we have the environment variable

  environment:
    - ON_DEMAND=True

But in my case, I'd like 2/4 cameras to be on-demand only

Thank you

**Comment by mrlt8** [comment]
Not right now. Is there any particular reason on-demand doesn't work for the the two cameras that need to be alway-on? They _should_ keep streaming as long as the client is connected.
I was planning to eventually invert the option so that it defaults to on-demand with always-on as an option.

**Comment by thatdaveguy1** [comment]
Oh, the opposite. The two always-on cameras work great.

But my Raspberry Pi is struggling with Wyze-Bridge + Homebridge + Scrypted. I was thinking if I made my 2 lesser used feeds on-demand, I could stop working my Raspberry so hard

**Reply by mrlt8** [reply]
Will see what I can do, but setting `ON_DEMAND=True` should allow all four to be on-demand and the two that you need always on should continue streaming to Scrypted if you have prebuffered enabled.

**Comment by thatdaveguy1** [comment]
Clever, but it didn't seem to work for some reason. Possibly too much CPU usage start-stopping the on-demand feeds

**Reply by mrlt8** [reply]
There might be a slight delay during startup as the bridge has to find and connect to the cameras, but it shouldn't be more CPU intensive than usual. Are you running the bridge on a raspberry pi?

**Comment by thatdaveguy1** [comment]
It's okay now! Since Wyzebridge doesn't like M1 Macs, I've changed my set up to:

Wyzebridge + Homebridge on RPI4 -> Scrypted on M1

Works great now. Scrypted is beastly heavy if I want HKSV, wipes out my poor RPI
